examples/src/main/java/io/kubernetes/client/examples/ParseExample.java

```diff
@@ -61,4 +66,31 @@ public static void main(String[] args) throws IOException, ApiException, ClassNo
 
     System.out.println(output);
   }
+
+  /**
+   * a example to mock "kubectl get configName -o json"
+   *
+   * @return @throws ApiException，IOException
+   */
+  public static void mockKubeJson() throws ApiException, IOException {
+
+    ApiClient client = Config.defaultClient();
+    Configuration.setDefaultApiClient(client);
+
+    CoreV1Api api = new CoreV1Api();
+    com.squareup.okhttp.Call call =
+        api.readNamespacedConfigMapCall("configmap-name", "default", null, null, null, null, null);
+
+    Response response = call.execute();
+
+    if (!response.isSuccessful()) {
+      return;
+    }
+
+    String body = response.body().string();
+
+    JsonObject returnData = new JsonParser().parse(body).getAsJsonObject();
```

Comment:
will this work with the custom-type adapter? in the existing example, it's parsing by using the nested gson instance where we register the adapters?

```java
    Object obj2 =
        client	        client
            .getJSON()	            .getJSON()
```

Reply:
Hrm, I think that Gson adapters were injected via annotation so that they would work everywhere, e.g. https://github.com/kubernetes-client/java/blob/master/kubernetes/src/main/java/io/kubernetes/client/custom/Quantity.java#L14

Reply:
https://github.com/kubernetes-client/java/blob/607a34b2dad0af5ea87ec6e37cf7db4536f545af/kubernetes/src/main/java/io/kubernetes/client/JSON.java#L46-L54

but here we're registering the adapters explicitly

Reply:
Ah, that's true for those built-ins we can't annotate...